### PR TITLE
upgrade to the new urql major

### PR DESCRIPTION
**Pull request by JoviDeCroock**
## Description

We have bumped our major, for types the biggest change is our new variables type https://github.com/FormidableLabs/urql/pull/2607

Related https://github.com/dotansimha/graphql-code-generator/issues/8276

Seeing is we see issues like https://github.com/FormidableLabs/urql/issues/2668 or https://github.com/FormidableLabs/urql/issues/2639 as well 😅 

## Type of change

Have left this in in case we see issues in the tests 😅 but should be good unless the new variables type includes issues, we have tested it on our side and it didn't but it still could

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How Has This Been Tested?

Ran locally on our application

## Checklist:

- [x] I have followed the [CONTRIBUTING](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md) doc and the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/plugins/typescript/urql-svelte-operations-store/package.json [hunk 1]
```diff
@@ -21,8 +21,7 @@
   },
   "peerDependencies": {
     "graphql": "^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0 || ^16.0.0",
-    "graphql-tag": "^2.0.0",
-    "@urql/exchange-graphcache": "^4.1.1"
```
Comment (JoviDeCroock) [inline packages/plugins/typescript/urql-svelte-operations-store/package.json:25]: Svelte shouldn't require graphcache

File: package.json [hunk 2]
```diff
@@ -51,7 +51,7 @@
     "@types/webpack-env": "1.18.0",
     "@typescript-eslint/eslint-plugin": "5.36.1",
     "@typescript-eslint/parser": "5.36.1",
-    "@urql/exchange-graphcache": "4.4.3",
+    "@urql/exchange-graphcache": "5.0.0",
```
Comment (JoviDeCroock) [inline package.json:54]: Upgraded these for the tests

**Review by dotansimha — APPROVED** [review]

File: packages/plugins/typescript/urql-svelte-operations-store/package.json (hunk 1 above)
Comment (dotansimha) [reply packages/plugins/typescript/urql-svelte-operations-store/package.json:25]: good catch, thanks! 
(fyi @jycouet )

**Comment by dotansimha** [comment]
Thanks @JoviDeCroock ! Will merge and release when CI clears :)

**Comment by JoviDeCroock** [comment]
If I can do anything about the failing linting check LMK, can't tell what's wrong atm 😅

**Comment by dotansimha** [comment]
> If I can do anything about the failing linting check LMK, can't tell what's wrong atm 😅

That's my bad, we can ignore for now :)

**Comment by dotansimha** [comment]
Thanks @JoviDeCroock ! Release is available here: 
https://github.com/dotansimha/graphql-code-generator/releases/tag/release-1662283159919

**Pull request merged**